<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add</class>
 <widget class="QDialog" name="add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding new film</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>25</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
  <widget class="QLabel" name="genre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>85</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Genre</string>
   </property>
  </widget>
  <widget class="QLabel" name="year">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>145</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Year</string>
   </property>
  </widget>
  <widget class="QLabel" name="likes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>205</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Likes</string>
   </property>
  </widget>
  <widget class="QLabel" name="trailer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>255</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Trailer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="title_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="genre_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="year_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="likes_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="trailer_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addfilm">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>350</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
